src/static/helpers/slasHelper.ts

```diff
@@ -292,13 +314,17 @@ export async function loginIDPUser(
     const optionsToken = {
       headers: {
         Authorization: authHeaderIdSecret,
+        ...options?.headers,
```

Comment:
I'm ok having this the headers spread here since `options?.headers` is a new argument, but we'll allow the possibility for developers to accidentally/incorrectly override `Authorization`

Reply:
what's the use case of overriding the `Authorization` header?

If you want to use a different clientid/secret, you should update the argument for the api client, not overriding it in slas helpers?

If you want to use a different format other than base64 client:secret encoding, does the API still work? If no, we should not allow it to be overriden.

Reply:
afaik, there is no other way then the base64 for authorization header. I've moved it above to avoid overriding for it. Thanks for the feedback